app/config/routing.yml

```diff
@@ -2,5 +2,10 @@ app:
     resource: "@AppBundle/Controller"
     type: annotation
 
+easy_admin_bundle:
+    resource: "@EasyAdminBundle/Controller/AdminController.php"
+    type:     annotation
+    prefix:   /admin
 
-
+facebook_logout:
+    path: /logout
```

Comment:
"no new line at the end of file", it could be configured in phpstorm to add it automatically, google it.